File: createJson/src/main/java/com/ruzhan/jsonfile/data/TheLieWeLive.java
package com.ruzhan.jsonfile.data;

import com.ruzhan.jsonfile.model.Introduce;
import com.ruzhan.jsonfile.model.Movie;
import com.ruzhan.jsonfile.model.MovieDetail;
import com.ruzhan.jsonfile.model.Video;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by ruzhan123 on 2018/7/3.
 */
public class TheLieWeLive {

    private static final int ID = 1103;
    private static final String PLAY_COUNT = "1";

    private static final String TITLE = "我们身处的谎言世界";
    private static final String TAG = "励志片";

    private static final String IMAGE = "https://raw.githubusercontent.com/ruzhan123/Lion/master/json/api/image/lie-we-live.jpg";

    private static final String DESC = "YouTube的2015年思想颠覆励志片，预设共28种语言，点击率达到八百多万次数。";

    public static final Movie movie = new Movie();
    public static final MovieDetail movieDetail = new MovieDetail();

    private static final List<Introduce> introduceList = new ArrayList<>();
    private static final List<Video> videoList = new ArrayList<>();

    static {
        introduceList.add(new Introduce(Introduce.TEXT,
                DESC, ""));

        introduceList.add(new Introduce(Introduce.IMAGE, "",
                "https://raw.githubusercontent.com/ruzhan123/Lion/master/json/api/image/lie-we-live01.jpg"));

        introduceList.add(new Introduce(Introduce.TEXT,
                "该片重点地完整引述了一系列社会问题丶环境保护丶大自然、全球化和阴谋论，以及未来展望对人类的觉醒作为剧终。", ""));

        videoList.add(new Video(String.valueOf(ID + 11),
                "我们身处的谎言世界", IMAGE, PLAY_COUNT, 1, "电影",
                "http://player.bilibili.com/player.html?aid=4472396&cid=7247511&page=1"));


        movie.id = String.valueOf(ID);
        movie.title = TITLE;
        movie.tag = TAG;
        movie.desc = DESC;
        movie.image = IMAGE;

        movieDetail.id = String.valueOf(ID + 3);
        movieDetail.movieId = String.valueOf(ID);
        movieDetail.title = TITLE;
        movieDetail.tag = TAG;
        movieDetail.desc = DESC;
        movieDetail.image = IMAGE;

        movieDetail.introduces = new ArrayList<>();
        movieDetail.introduces.addAll(introduceList);

        movieDetail.videos = new ArrayList<>();
        movieDetail.videos.addAll(videoList);
    }
}

File: createJson/src/main/java/com/ruzhan/jsonfile/data/TheLieWeLive.kt
package com.ruzhan.jsonfile.data

import com.ruzhan.jsonfile.model.Introduce
import com.ruzhan.jsonfile.model.Movie
import com.ruzhan.jsonfile.model.MovieDetail
import com.ruzhan.jsonfile.model.Video

import java.util.ArrayList

object TheLieWeLive {

    private const val ID = 1103
    private const val PLAY_COUNT = "1"

    private const val TITLE = "我们身处的谎言世界"
    private const val TAG = "社会·励志片"

    private const val IMAGE = "https://raw.githubusercontent.com/ruzhan123/Lion/master/json/api/image/lie-we-live.jpg"

    private const val DESC = "YouTube的2015年思想颠覆励志片，预设共28种语言，点击率达到八百多万次数。"

    val movie = Movie()
    val movieDetail = MovieDetail()

    private val introduceList = ArrayList<Introduce>()
    private val videoList = ArrayList<Video>()

    init {
        introduceList.add(Introduce(Introduce.TEXT,
                DESC, ""))

        introduceList.add(Introduce(Introduce.IMAGE, "",
                "https://raw.githubusercontent.com/ruzhan123/Lion/master/json/api/image/lie-we-live01.jpg"))

        introduceList.add(Introduce(Introduce.TEXT,
                "该片重点地完整引述了一系列社会问题丶环境保护丶大自然、全球化和阴谋论，以及未来展望对人类的觉醒作为剧终。", ""))

        videoList.add(Video((ID + 11).toString(),
                "我们身处的谎言世界", IMAGE, PLAY_COUNT, 1, "电影",
                "http://player.bilibili.com/player.html?aid=4472396&cid=7247511&page=1"))


        movie.id = ID.toString()
        movie.title = TITLE
        movie.tag = TAG
        movie.desc = DESC
        movie.image = IMAGE

        movieDetail.id = (ID + 3).toString()
        movieDetail.movieId = ID.toString()
        movieDetail.title = TITLE
        movieDetail.tag = TAG
        movieDetail.desc = DESC
        movieDetail.image = IMAGE

        movieDetail.introduces = ArrayList()
        movieDetail.introduces.addAll(introduceList)

        movieDetail.videos = ArrayList()
        movieDetail.videos.addAll(videoList)
    }
}
